packages/agoric-cli/src/commands/oracle.js

```diff
@@ -297,43 +296,59 @@ export const makeOracleCommand = (logger, io = {}) => {
           show(fmtFeed(before));
         }
 
-        console.error(
-          'Choose lead oracle operator order based on latestRound...',
-        );
         const keyOrder = keys.map(normalizeAddress);
-        const latestRoundP = readLatestHead(
-          `published.priceFeed.${pair[0]}-${pair[1]}_price_feed.latestRound`,
-        );
-        await Promise.race([
-          delay(5000),
-          latestRoundP.then(round => {
-            // @ts-expect-error XXX get type from contract
-            const { roundId, startedAt, startedBy } = round;
-            show({
-              startedAt: fmtSecs(startedAt.absValue),
-              roundId,
-              startedBy,
-            });
-            if (startedBy === keyOrder[0]) {
-              keyOrder.reverse();
-            }
-          }),
-        ]).catch(err => {
-          console.warn(err);
-        });
+        if (before) {
+          console.error(
+            'Choose lead oracle operator order based on latestRound...',
+          );
 
-        const instance = lookupPriceAggregatorInstance(pair);
+          const latestRoundP =
+            /** @type {Promise<{roundId: number, startedAt: import('@agoric/time').TimestampRecord, startedBy: string}>} */ (
+              readLatestHead(
+                `published.priceFeed.${pair[0]}-${pair[1]}_price_feed.latestRound`,
+              )
+            );
+          await Promise.race([
+            delay(5000),
+            latestRoundP.then(round => {
+              const { roundId, startedAt, startedBy } = round;
+              show({
+                startedAt: fmtSecs(startedAt.absValue),
+                roundId,
+                startedBy,
+              });
+              if (startedBy === keyOrder[0]) {
+                keyOrder.reverse();
+              }
+            }),
+          ]).catch(err => {
+            console.warn(err);
+          });
+        }
 
-        console.error('pushPrice from each:', keyOrder);
+        const instance = lookupPriceAggregatorInstance(pair);
+        const adminOfferIds = {};
         for await (const from of keyOrder) {
-          const oracleAdminAcceptOfferId = await findOracleCap(
+          adminOfferIds[from] = await findOracleCap(
             instance,
             from,
             readLatestHead,
           );
-          if (!oracleAdminAcceptOfferId) {
-            throw Error(`no oracle invitation found: ${from}`);
+          if (!adminOfferIds[from]) {
+            console.error('no oracle invitation found: ', from);
```

Comment:
This tool isn't actually looking for an invitation. Perhaps this would be more clear:

```suggestion
            console.error('failed to find previous offer from accepting oracle invitation: ', from);
```